x-pack/docs/en/rest-api/security/get-api-keys.asciidoc

```diff
@@ -0,0 +1,118 @@
+[role="xpack"]
+[[security-api-get-api-key]]
+=== Get API Key information API
+++++
+<titleabbrev>Get API key information</titleabbrev>
+++++
+
+Retrieves information for one or more API keys.
+
+==== Request
+
+`GET /_security/api_key`
+
+==== Description
+
+The information for the API keys created by <<security-api-create-api-key,create API Key>> can be retrieved
+using this API.
+
+==== Request Body
+
+The following parameters can be specified in the query parameters of a GET request and
+pertain to retrieving api keys:
+
+`id` (optional)::
+(string) An API key id. This parameter cannot be used with any of `name`, `realm_name` or
+         `username` are used.
+
+`name` (optional)::
+(string) An API key name. This parameter cannot be used with any of `id`, `realm_name` or
+                          `username` are used.
+
+`realm_name` (optional)::
+(string) The name of an authentication realm. This parameter cannot be used with either `id` or `name`.
+
+`username` (optional)::
+(string) The username of a user. This parameter cannot be used with either `id` or `name`.
+
+NOTE: While all parameters are optional, at least one of them is required.
+
+==== Examples
+
+The following example to retrieve the API key identified by specified `id`:
+
+[source,js]
+--------------------------------------------------
+GET /_security/api_key?id=dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==
+--------------------------------------------------
+// NOTCONSOLE
```

Comment:
at some point we need to go back and add tests for these docs in SecurityDocumentationIT